scripts/scripts-1.js

```diff
@@ -0,0 +1,90 @@
+for (let i = 10; i < 26; i += 0.5) {
+
+  console.log(i);
+}
+
+const number = +prompt("Введіть число"); {
+
+  if (number % 2 !== 0 && number % 3 !== 0) {
+
+    console.log(`${number} просте число`);
+  }
+
+  else {
+    console.log(`${number} складене число`);
+  }
+}
+
+localStorage.setItem("number", number);
+
+discount = +prompt("Сума вашої покупки становить:");
```

Comment:
```suggestion
const discount = +prompt("Сума вашої покупки становить:");
```
Also name of the variable does not reflect the value `discount` it's about the discount amount but not for the purchase sum